config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gke.yaml

```diff
@@ -51,6 +54,36 @@ periodics:
       - --provider=gke
       - --skew
       - --timeout=1200m
+      - --test_args=--ginkgo.skip=\[Slow\]
```

Comment:
just focus on serial + disruptive?

Reply:
also this will.. include all Features tag which you don't want

maybe reference https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml#L78-L129

we can do an overhaul for all upgrade jobs.